pages.fr/common/cmake.md

```diff
@@ -0,0 +1,28 @@
+# cmake
+
+> Système de construction logicielle multiplateforme, qui permet de générer des recettes de construction pour les systèmes de construction natifs.
+> Plus d'informations : <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
+
+- Générer une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```

Comment:
Please use présent de l'indicatif
```suggestion
- Génére une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```

Reply:
```suggestion
- Génère une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```

Reply:
Sorry I kind of mixed up infinitive/imperative (yeah you already told me about this 😅), but it is fixed now

Reply:
Don't worry. :+1: